gtk2/plan.sh

```diff
@@ -0,0 +1,66 @@
+pkg_name=gtk2
+pkg_origin=core
+pkg_version=2.24.31
+pkg_description="GTK+, or the GIMP Toolkit, is a multi-platform toolkit for creating graphical user interfaces."
+pkg_upstream_url="https://www.gtk.org"
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=('LGPL-2.0')
+upstream_name="gtk+"
+pkg_source="https://download.gnome.org/sources/${upstream_name}/$(echo $pkg_version | cut -d. -f1-2)/${upstream_name}-${pkg_version}.tar.xz"
```

Comment:
`$(echo $pkg_version | cut -d. -f1-2)` is better written as `${pkg_version%.*}`.